crates/sui-types/src/quorum_driver_types.rs

```diff
@@ -44,4 +44,9 @@ pub enum QuorumDriverError {
     NonRecoverableTransactionError {
         errors: Vec<(SuiError, Vec<AuthorityName>, StakeUnit)>,
     },
+    #[error("Transaction is not processed because {overloaded_stake} of validators by stake are overloaded with ceritificates pending execution.")]
```

Comment:
```suggestion
    #[error("Transaction is not processed because {overloaded_stake} of validators by stake are overloaded with certificates pending execution.")]
```